packages/@sanity/default-layout/src/navbar/search/contexts/commandList/index.tsx

```diff
@@ -202,6 +202,34 @@ export function CommandListProvider({
     []
   )
 
+  const scrollToNextItem = useCallback(() => {
```

Comment:
Might be overkill/require passing to many variables, but is there a useScrollTo('next' | 'previous') abstraction hiding here that would eliminate all duplication execpt selecting +1 | 1 for moving the index?